### PR TITLE
Sanity PR Attempt

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
Shrimple - ports over Eris' working sanity system.

- New perks have been added from Eris, such as Wealthy Upbringing background now giving the Noble perk, Rough Life incorporating parts of the Vagabond perk, Survivor added as an oddity perk, etc.
- Pet-mobs now restore sanity by holding them or having them on you.
- Church obelisk perk no longer gives you health boosts. Now - it restores sanity massively instead for Church people.
- Masks, like surgical and gas, prevent sanity loss from bad enviroments
- Psi-mobs in deep maint should tank sanity down.
- Smoking will increase sanity as well as drinking alchy-hol (:

Tl;dr - possum approved
image

## Changelog
:cl:
adds: sanity back to the game and keeps us mostly up-to-date with all Eris sanity related systems and perks.
/:cl:

### CODE REVIEW

**Comment by Stutternov** [comment]
It lives.
